apps/extension/src/Approvals/ApproveConnection.tsx

```diff
@@ -10,17 +10,12 @@ export const ApproveConnection: React.FC = () => {
   const requester = useRequester();
   const params = useQuery();
   const interfaceOrigin = params.get("interfaceOrigin");
-  const interfaceTabId = params.get("interfaceTabId");
```

Comment:
`interfaceTabId` wasn't used anymore, it was replaced by `popupTabId` directly on the service